node/node.go

```diff
@@ -91,7 +91,7 @@ func NewNode(genDoc *genesis.Genesis, conf *config.Config,
 	if conf.GRPC.BasicAuth != "" {
 		enableHTTPAuth = true
 	}
-	grpcServer := grpc.NewServer(conf.GRPC, st, syn, net, consMgr, walletMgr)
+	grpcServer := grpc.NewServer(conf.GRPC, st, syn, net, consMgr, walletMgr, conf.TxPool)
```

Comment:
@b00f What you think we pass just `conf` instead part of configuration?

Reply:
Nope! We can't add parameter that simple!
Try to find another way...

Reply:
Let's think out of box; gRPC almost have everything. Right?
Why don't we simply give him Node.

Reply:
We can give access to configuration via node?

Reply:
node itself depends on gRPC

Reply:
Node has all modules...

Reply:
If we pass node to gRPC we will encounter **cycle import**.